rxpager/src/test/java/com/schibstedspain/android/rxpager/PagerTest.java

```diff
@@ -34,131 +36,125 @@
 
   private Pager<TokenPage<String>, String> pager;
   @Mock
-  private Func1<String, Observable<TokenPage<String>>> getPageMock;
+  private Function<String, Observable<TokenPage<String>>> getPageMock;
 
   @Before
   public void setUp() {
-    pager = new Pager<>(null, (s, stringTokenPage) -> stringTokenPage.getNextPageToken(), getPageMock);
+    pager = new Pager<>("0", (s, stringTokenPage) -> stringTokenPage.getNextPageToken(), getPageMock);
```

Comment:
I guess `""` would be more appropriate, you don't know what token could use the consumer of the library.
Also I'm thinking ( not forcing you to do it now ) that maybe we should expose this detail in the readme.